### ISSUE TITLE
[MHV APPT / Accessibility] Breadcrumb focus issue on Appointments

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#130528]
### 📝 User story

Apply the fix from https://github.com/department-of-veterans-affairs/va.gov-team/issues/123109 to the MHV Appointments product.

### 📝 Problem statement

See https://github.com/department-of-veterans-affairs/va.gov-team/issues/123109

### 👣 Steps to reproduce the issue

See: https://github.com/department-of-veterans-affairs/va.gov-team/issues/123109

### 🔗 Point of contact

@steveg-IAT

### ⚠️ Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### 📝 Related issues

Parent issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/123109

### 🔎 Accessibity standards

see: https://github.com/department-of-veterans-affairs/va.gov-team/issues/123109

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to "QA/A11y Review" after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by outerpress** [comment]
Thanks @sarahhortonadhoc ! @simiadebowale @ala-yna fya, I'll bring this up as a PL at our next standup to discuss

**Comment by sarahhortonadhoc** [comment]
Looks great, great work! 🥳
